pages/Documents/Reporting/OutboundReportingAPI/overview.md

```diff
@@ -166,17 +167,26 @@ Click [**Account**](https://proactive-messaging.z1.fs.liveperson.com/api/api-doc
 
 ```json
 {
- "filters": {
-   "channels": [
-     "sms"
-   ],
-   "skills": [
-     "billing"
-   ]
- }
+ "channels": [
+   "sms"
+ ],
+ "skills": [
+   "billing"
+ ],
+ "messagestatus": [
+   "DELIVERED", "FAILED", "RESPONDED", "READ"
+ ],
+ "handoffids": [
+   "H123456", "H987655"
+ ],
+  "source": [
+   "API", "UI", "LPWF"
+ ]
 }
 ```
 
+Each of the filter options shown above are optional. Regardless of what filter options are used, the results will always be grouped by channel, skill, and transactionday. When additional filters, such as source or messagstatus or handoffid are added, the results will be grouped by them as well and these will fields will be included in the response.
```

Comment:
@jschatzman-lp the second sentence is breaking and needs to be corrected.

Reply:
@lp-gdoshi Fixed